src/main/java/net/sf/jabref/logic/formatter/Formatters.java

```diff
@@ -45,7 +45,7 @@
 public class Formatters {
 
     public static final List<Formatter> CONVERTERS = Arrays.asList(
-            new HtmlToLatexFormatter(),
+            new HtmlToLatexFormatter(false), // TODO: Get the correct value from USE_CONVERT_TO_EQUATION
```

Comment:
Maybe we should keep the Globals dependency here for now (`new HtmlToLatexFormatter(Globals.prefs.USE_CONVERT_TO_EQUATION)`), because this changes the behavior otherwise.